src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -520,9 +520,16 @@ class LitShader {
             code += "    gl_FragColor = packFloat(depth);\n";
         } else if (!isVsm) {
             code += "    gl_FragColor = vec4(1.0);\n"; // just the simplest code, color is not written anyway
+            const exportDepth = shadowType === SHADOW_PCF1 || shadowType === SHADOW_PCF3 || shadowType === SHADOW_PCF5 ||
```

Comment:
do we need to handle this differently for WebGl1?
Also, just realised, we'd need to test this (including PCSS) on other iOS devices like iPhone 6s I think that use WebGL2 but I'm not sure they can render to Float textures, but can render to half float, of the other way around, not sure now.

Reply:
I think for GL1 we would enter the packed depth flow.

Reply:
Can you please confirm this and test all shadow options you can on gl1 and 2